README.md

```diff
@@ -78,9 +78,9 @@ SteamTinkerLaunch can be installed in a few different ways depending on your pla
-| [ProtonUp-Qt](https://github.com/sonic2kk/steamtinkerlaunch/wiki/ProtonUp-Qt) | As of v2.7.3, [ProtonUp-Qt](https://github.com/DavidoTek/ProtonUp-Qt) has support for SteamTinkerLaunch. This should allow you to install SteamTinkerLaunch regardless which distribution you are using **including Steam Deck**. See the [Installation wiki](https://github.com/sonic2kk/steamtinkerlaunch/wiki/Installation#protonup-qt) and our [ProtonUp-Qt wiki page](https://github.com/sonic2kk/steamtinkerlaunch/wiki/ProtonUp-Qt) for more details.<br /><br />Outside of Steam Deck, ensure you have met the relevant SteamTinkerLaunch [hard dependencies](https://github.com/sonic2kk/steamtinkerlaunch/wiki/Installation#hard-dependencies). |
-| Manual Installation | SteamTinkerLaunch supports **system-wide** (root) and **local** (non-root) manual installation. [See Installation Wiki notes](https://github.com/sonic2kk/steamtinkerlaunch/wiki/Installation#manual-installation) for setup and details. |
+| Manual Installation | Most up to date and therefore preferred installation method. Requires getting some dependencies and (if doing a root installation) installing from the Makefile. SteamTinkerLaunch supports **system-wide** (root) and **local** (non-root) manual installation. [See Installation Wiki notes](https://github.com/sonic2kk/steamtinkerlaunch/wiki/Installation#manual-installation) for setup and details. |
+| Package Manager | See distribution package status below, though this list may not be exhaustive.<br />Many thanks to all package maintainers!<br />(Keep in mind that a green status does not necessarily imply up to date; this is an artifact of Repology. As of around June 9th 2024, the most up to date version is `steamtinkerlaunch-v14.0.20240609-3`.)<br /><br />[![Packaging status](https://repology.org/badge/vertical-allrepos/steamtinkerlaunch.svg)](https://repology.org/project/steamtinkerlaunch/versions)<br /><br />Refer to the [Installation Wiki](https://github.com/sonic2kk/steamtinkerlaunch/wiki/Installation#package-manager) for more information on available packages. |
+| [ProtonUp-Qt](https://github.com/sonic2kk/steamtinkerlaunch/wiki/ProtonUp-Qt) | As of v2.7.3, [ProtonUp-Qt](https://github.com/DavidoTek/ProtonUp-Qt) has support for SteamTinkerLaunch. This should allow you to install SteamTinkerLaunch regardless which distribution you are using **including Steam Deck**. *Keep in mind that to get the most up to date (master) version of SteamTinkerLaunch, you will need to enable `Advanced Mode`.* See the [Installation wiki](https://github.com/sonic2kk/steamtinkerlaunch/wiki/Installation#protonup-qt) and our [ProtonUp-Qt wiki page](https://github.com/sonic2kk/steamtinkerlaunch/wiki/ProtonUp-Qt) for more details.<br /><br />Outside of Steam Deck, ensure you have met the relevant SteamTinkerLaunch [hard dependencies](https://github.com/sonic2kk/steamtinkerlaunch/wiki/Installation#hard-dependencies). |
```

Comment:
```suggestion
| [ProtonUp-Qt](https://github.com/sonic2kk/steamtinkerlaunch/wiki/ProtonUp-Qt) | As of v2.7.3, [ProtonUp-Qt](https://github.com/DavidoTek/ProtonUp-Qt) has support for SteamTinkerLaunch. This should allow you to install SteamTinkerLaunch regardless which distribution you are using **including Steam Deck**.<br /><br/>You should enable ProtonUp-Qt's `Advanced Mode` which allows you to install `SteamTinkerLaunch-git` (recommended, latest master branch). See the [Installation wiki](https://github.com/sonic2kk/steamtinkerlaunch/wiki/Installation#protonup-qt) and our [ProtonUp-Qt wiki page](https://github.com/sonic2kk/steamtinkerlaunch/wiki/ProtonUp-Qt) for more details.<br /><br />Outside of Steam Deck, ensure you have met the relevant SteamTinkerLaunch [hard dependencies](https://github.com/sonic2kk/steamtinkerlaunch/wiki/Installation#hard-dependencies). |
```

This is mostly just stylistic/wording preference here. Feel free to push back if you feel strongly enough about it, it's all the same information you have included, but arranged a bit differently from the way I would prefer it to be read if it was me.

Arguably the backticks around `Advanced Mode` aren't "correct" but I prefer them as you have them, although I'd prefer if the text wasn't italicized.

If you have other wording/formatting suggestions off of this though feel free to suggest as well!